src/intl/messages.js

```diff
@@ -62,6 +62,13 @@ export const messages: { [messageId: string]: MessageType } = {
   cancel: 'Cancel',
   cannotUpdateCloudInitHostname: 'Host name cannot be synchronized with VM name because the VM name is not a valid host name.',
   cantLogonToConsole: 'Single sign on failed. Please check to see if the guest agent is running on your virtual machine. Contact your administrator if the problem continues.',
+  cantOpenConsole: "Can't open console: {message}",
+  cantOpenGraphicalConsoleEmptyList: "Can't open graphical console since {vmName} is a headless VM.",
+  cantOpenGraphicalConsoleMoreOptions: "Can't open graphical console you can find more options in the VM page.",
+  cantOpenGraphicalConsoleNoDefault: "Can't open graphical console {vmName} has no {defaultConsole} console assigned. You can find more options in the VM page.",
+  cantOpenNonRdpGraphicalConsoleEmptyList: "Can't open non Rdp graphical console since {vmName} is a headless VM.",
+  cantOpenNonRdpGraphicalConsoleMoreOptions: "Can't open non Rdp graphical console you can find more options in the VM page.",
+  cantOpenNonRdpGraphicalConsoleNoDefault: "Can't open non Rdp graphical console {vmName} has no {defaultConsole} console assigned. You can find more options in the VM page.",
```

Comment:
relevant to all new added console messages:
1. "You can find more options in the VM page" - what do you mean by that? Which info on which page?
2.Why do we need those "non RDP" message? Console is the same regardless to RDP.

Reply:
>1. "You can find more options in the VM page" - what do you mean by that? Which info on which page?

I meant to inform users that in case of no default they can find more options like: RDP, another console in the vm details page.
>2. Why do we need those "non RDP" messages? Console is the same regardless to RDP.

It's just mention that RDP is available, I'll remove that...